napari/layers/tracks/tracks.py

```diff
@@ -333,7 +338,7 @@ def _update_thumbnail(self):
             colormapped[coords[:, 1], coords[:, 0]] = colors
 
         colormapped[..., 3] *= self.opacity
-        self.thumbnail = colormapped
+        self.thumbnail = colormapped.astype(np.uint8)
```

Comment:
I think this line is causing at least one of the test failures. Given thumbnails should always be uint8 data type, I think this has revealed a bug in the failing test?

I don't have time to investigate further or update this PR now, but if anyone else wants to feel free to push to this branch/finish off the PR.

Reply:
The problem is caused by `nan` in a `colormapped` array. I do not know how to handle them. Replace by 0? 

@napari/core-devs

Reply:
This is just for the thumbnail? I would say putting `0` is totally fine there.